docs/faq/index.rst

```diff
@@ -86,5 +46,23 @@ Your message will be forwarded to the ClusterHQ security team (a small group of
 Due to the sensitive nature of security issues, we ask you not to send a message to one of the public mailing lists.
 ClusterHQ has a policy for :ref:`reporting-security-issues` designed to minimize any damage that could be inflicted through public knowledge of a defect while it is still outstanding.
 
+ZFS
+~~~
+
+Flocker uses ZFS in experimental configurations. What about the ZFS licensing issues?
```

Comment:
I think this may as well be shortened to "What about the ZFS licensing issues?"

And add a ``:note:`` above saying "Flocker's ZFS backend is experimental" ...or a message that is consistent with similar messages elsewhere in the docs.